windows/src/robocorp/windows/__init__.py

```diff
@@ -114,13 +114,50 @@ def config() -> "Config":
     return __get_cache_mod().config()
 
 
+@overload
+def find_window(
+    locator: Locator,
+    search_depth: int = ...,
+    timeout: Optional[float] = ...,
+    wait_time: Optional[float] = ...,
+    foreground: bool = ...,
+    raise_error: Literal[True] = ...,
+) -> "WindowElement":
+    ...
+
+
+@overload
+def find_window(
+    locator: Locator,
+    search_depth: int = ...,
+    timeout: Optional[float] = ...,
+    wait_time: Optional[float] = ...,
+    foreground: bool = ...,
+    raise_error: Literal[False] = ...,
+) -> Optional["WindowElement"]:
+    ...
+
+
+@overload
+def find_window(
+    locator: Locator,
+    search_depth: int = ...,
+    timeout: Optional[float] = ...,
+    wait_time: Optional[float] = ...,
+    foreground: bool = ...,
+    raise_error: bool = ...,
```

Comment:
The `@overload` needs to happen for the variable type value (`bool`) as well to please mypy.